templates/vars/vnet/ithc.json

```diff
@@ -56,6 +56,28 @@
           "service": "Microsoft.Storage"
         }
       ]
-    }
+    },
+    "routetableRoutes": {
+      "value": [
+          {
+          "name": "AKSITHCtoHUBPalo",
+          "nextHopType": "VirtualAppliance",
+          "nextHopIpAddress": "10.11.72.36",
+          "addressPrefix": "10.0.0.0/8"
+          },
+          {
+            "name": "BastionMgmtVnet",
```

Comment:
?

Reply:
Will remove these BastionMgmtVnet for now, they were added I assume manually at some stage prior.